### PR TITLE
Improve swagger doc

### PR DESCRIPTION
**Pull request by sapk**
Response type and method params are in sdk.

Needed for go-gitea/gitea#2274

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

File: gitea/hook.go [hunk 1]
```diff
@@ -31,6 +32,10 @@ type Hook struct {
 	Created time.Time         `json:"created_at"`
 }
 
+// HookList represents a list of API hook.
+// swagger:response HookList
+type HookList []*Hook
+
 // ListOrgHooks list all the hooks of one organization
 func (c *Client) ListOrgHooks(org string) ([]*Hook, error) {
```
Comment (lunny) [inline gitea/hook.go]: Should this return param is `HookList` ?

Comment (sapk) [reply gitea/hook.go]: Yes it is better to do it.

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**